### ISSUE TITLE
How to handle child collections of preserved collections

### ISSUE THREAD

**Issue by bondjimbond** [bondjimbond/islandora_westvault#23]
Once #20 is merged, we'll have the option to automatically apply a PRESERVATION datastream to all child objects of a collection.

Problem: what to do when one of these children is also a collection?

Options:
- Don't apply PRESERVATION to collections automatically; admins must decide about preservation per-collection
- Create a PRESERVATION datastream to child collections, with the same `<preservationMethod>` as their parents
- Some third approach

**Comment by bondjimbond** [comment]
Noted in #27: 

> Future development could allow child collections to inherit the preservationMethod of the parent and run the loop on all their children as well. This is a first step - stops child collections from getting a PRESERVATION datastream without any functionality attached.

How would that look:
- Checkbox or radios detailing how to handle child collections
- Move the [collection_children loop](https://github.com/bondjimbond/islandora_westvault/blob/7.x/includes/management_form.inc#L74-L80) into its own function so that it can be repeated

I would consider this nice-to-have and not a core feature... What do you think @mjordan?
